loadflow/loadflow-validation/src/main/java/com/powsybl/loadflow/validation/ValidationConfig.java

```diff
@@ -71,9 +69,7 @@ public static ValidationConfig load(PlatformConfig platformConfig) {
             ModuleConfig config = platformConfig.getModuleConfig("loadflow-validation");
             threshold = config.getDoubleProperty("threshold", THRESHOLD_DEFAULT);
             verbose = config.getBooleanProperty("verbose", VERBOSE_DEFAULT);
-            if (config.hasProperty("load-flow-factory")) {
-                loadFlowFactory = config.getClassProperty("load-flow-factory", LoadFlowFactory.class, componentDefaultConfig.findFactoryImplClass(LoadFlowFactory.class));
-            }
+            loadFlowName = config.getOptionalStringProperty("load-flow-name").orElse(null);
```

Comment:
Don't forget to update this [page](http://www.powsybl.org/docs/configuration/modules/loadflow-validation.html)

Reply:
done